README.md

```diff
@@ -129,6 +157,18 @@ The `is_final` field is always `True` because no updates will be output for the
 
     Example "captions" responses can be found in [examples/responses/captions.md](https://github.com/verbit-ai/verbit-streaming-python-sdk/blob/main/examples/responses/captions.md).
 
+### Error handling and recovery
+
+#### Initial connection
+In case the WebSocket client fails to establish the (initial) connection with the service (e.g. due to temporary unavailability), 
+it will perform exponential retry, up to `max_connection_retry_seconds` (configurable).
```

Comment:
indicate the name of the property through which this is configurable

Reply:
that's the property name, maybe we can hyperlink this soon to some Swagger gen pages, etc.

Reply:
Added a link to the property in that place.